contrib/win32/win32compat/console.c

```diff
@@ -627,16 +627,14 @@ ConWriteString(char* pszString, int cbString)
 	if ((needed = MultiByteToWideChar(CP_UTF8, 0, pszString, cbString, NULL, 0)) == 0 ||
 	    (utf16 = malloc(needed * sizeof(wchar_t))) == NULL ||
 	    (cnt = MultiByteToWideChar(CP_UTF8, 0, pszString, cbString, utf16, needed)) == 0) {
-		const char* pszStringConst = pszString;
-		Result = (DWORD)printf_s(pszStringConst);
+		Result = (DWORD)printf_s(pszString);	// CodeQL: [SM01734] false positive: call is not format string with arguments.
```

Comment:
nitpick - not sure how strict the suppression filter is 
```suggestion
		Result = (DWORD)printf_s(pszString);	// CodeQL [SM01734] false positive: call is not format string with arguments.
```